Jenkinsfile

```diff
@@ -106,27 +57,8 @@ pipeline {
                 pattern: 'build_*.log',
                 reportEncoding: 'UTF-8'
               ),
-              groovyScript(
-                parserId:'clippy-warnings',
-                pattern: 'clippy_*.log',
-                reportEncoding: 'UTF-8'
-              ),
-              groovyScript(
-                parserId:'clippy-errors',
-                pattern: 'clippy_*.log',
-                reportEncoding: 'UTF-8'
-              )
           ]
         )
-        script {
-          try {
-            sh './ci/check_fmt_log.sh'
```

Comment:
I think with this change, the whole file `check_fmt_log.sh` is obsolete now. If so, can you delete it?